src/main/java/uk/gov/ons/ctp/response/action/ActionSvcApplication.java

```diff
@@ -177,22 +174,10 @@ public PubSubInboundChannelAdapter actionCaseNotificationAdapter(
     PubSubInboundChannelAdapter adapter =
         new PubSubInboundChannelAdapter(pubSubTemplate, subscriptionName);
     adapter.setOutputChannel(inputChannel);
+    adapter.setAckMode(AckMode.MANUAL);
     return adapter;
   }
 
-  @Bean
-  @InboundChannelAdapter(
```

Comment:
It's still using a DirectChannel instead of PublishSubscribeChannel.   Is that  intentional?

Reply:
It shouldn't matter as its default. But to be consistent I will change it to PublishSubscribeChannel. Its good to have it consistent.